code/drasil-code/lib/Language/Drasil/Code/Imperative/Generator.hs

```diff
@@ -150,6 +150,10 @@ genPackage unRepr = do
         (foldlSent $ purpose $ codeSpec g)
       bckgrnd = show $ sentenceDoc db Implementation OneLine
         (foldlSent $ background $ codeSpec g)
+      mtvtn = show $ sentenceDoc db Implementation OneLine
```

Comment:
In that same refactor, I think all the stuff that concerns `README` generation should be moved out of this file. It could go into `Language/Drasil/Code/Imperative/Generator/README`.